scripts/performance/memory-performance/compute-memory-performance.js

```diff
@@ -0,0 +1,115 @@
+const puppeteer = require('puppeteer')
+const { timeout } = require('../../lib/execution-utils')
+const { fetchPR, LOCAL_BRANCH } = require('../../lib/git-utils')
+const NUMBER_OF_RUNS = 40 // Rule of thumb: 30 runs should be enough to get a good average
+const TASK_DURATION = 1000
+const ACTION_NAMES = [
+  'RUM - add global context',
+  'RUM - add action',
+  'RUM - add error',
+  'RUM - add timing',
+  'RUM - start view',
+  'RUM - start/stop session replay recording',
+  'Logs - log message',
+]
```

Comment:
**💬 suggestion:** ‏Instead of running a bunch of regexes on those names, you could simply hardcode the various values, like:

```js
const TESTS = [
  {
    name: 'RUM - add global context',
    button: '#add-global-context',
    property: 'add_global_context',
  }
]